main.py

```diff
@@ -162,74 +285,113 @@ def execute():
+                    ))
+                ).click()
+
             except TimeoutException:
                 logger.debug('no mature content block to bypass')
 
             logger.debug('find the purchase button')
             purchase_button = WebDriverWait(browser, TIMEOUT).until(
-                EC.visibility_of_element_located((By.XPATH, "//button[contains(@class,'Purchase')]"))
+                EC.visibility_of_element_located((
+                    By.XPATH,
+                    "//button[@data-testid='purchase-cta-button']"
+                ))
             )
 
             # name of the game
             logger.debug('extract game title')
-            name = browser.find_element_by_xpath("//h1[contains(@class,'NavigationVertical')]").text
+            name = browser.title
 
             # price formatted as '£11.99'
             logger.debug('extract game price')
             price = browser.find_element_by_xpath("//s").text
 
             # date formatted as 'Sale ends 11/29/2019 at 3:59 PM'
             logger.debug('extract sales end date')
-            expires = browser.find_element_by_xpath("//span[contains(text(),'Sale ends')]").text
+            expires = browser.find_element_by_xpath(
+                "//span[contains(text(),'Sale ends')]"
+            ).text
 
             if purchase_button.text == 'OWNED':
-                logger.info('\"%s\" already owned. Price was %s and %s', name, price, expires)
+                logger.info(
+                    '\"%s\" already owned. Price was %s and %s',
+                    name, price, expires
+                )
+
             elif purchase_button.text == 'GET':
                 logger.info('obtaining \"%s\"', name)
                 purchase_button.click()
 
                 purchase_steps(browser)
 
-                logger.info('obtained %s. Price was %s and %s', name, price, expires)
+                logger.info(
+                    'obtained %s. Price was %s and %s',
+                    name, price, expires
+                )
+
             elif purchase_button.text == 'SEE EDITIONS':
                 logger.debug('processing editions for \"%s\"', name)
                 # used to know the number of elements and titles
                 # without having to query on every refresh
                 editions_addons_titles = WebDriverWait(browser, TIMEOUT).until(
                     EC.visibility_of_all_elements_located((
-                        By.XPATH, "//div[contains(@class,'Editions-title') or contains(@class, 'AddOns-title')]"
-                    )))
-                editions_addons_titles = [i.text for i in editions_addons_titles]
+                        By.XPATH, "//div[contains(@class,'Editions-title') "
+                                  "or contains(@class, 'AddOns-title')]"
+                    ))
+                )
+
+                editions_addons_titles = [i.text for i in
+                                          editions_addons_titles]
```

Comment:
This one makes it very weird, if you really want to shorten the line rename the variable instead